File: src/main/java/com/giant/auth/dto/SignInDto.java
package com.giant.auth.dto;

public record SignInDto(
        Long accountId,
        String passwordHash,
        Long accountRoleId,
        String accountRoleName,
        String employeeCode,
        String employeeName,
        String employeeRoleName,
        String departmentName,
        String teamName,
        String positionName
) {
}

File: src/main/kotlin/com/giant/auth/dto/SignInDto.kt
package com.giant.auth.dto

data class SignInDto (
    val accountId: Long,
    val passwordHash: String,
    val accountRoleId: Long,
    val accountRoleName: String,
    val employeeRoleName: String,
    val departmentName: String,
    val teamName: String
)